unobstructed_lanechange/src/unobstructed_lanechange.cpp

```diff
@@ -611,4 +648,24 @@ namespace unobstructed_lanechange
         
     }
 
+    bool UnobstructedLaneChangePlugin::validate_yield_plan(const cav_msgs::TrajectoryPlan& yield_plan)
+    {
+        if (yield_plan.trajectory_points.size()>= 2)
+        {
+            if (yield_plan.trajectory_points[0].target_time > ros::Time::now())
```

Comment:
Is this guaranteed to be the case? I'd think yield plugins plan would start at `ros::Time::now()` when it generated the plan, which would be less than the `ros::Time::now()` value here.